### PR TITLE
Reviewed qgis QuickStart

### PR DESCRIPTION
**Pull request by flicstar**
Reviewed as part of the QuickStart project during Google Season of Docs 2019. The aim is consistency across all OSGeoLive QuickStarts.
The document is structured well and only requires minor cosmetic and wording changes.
@dakcarto  Please action comments and suggestions you agree with, and ignore those you don't.

Partner trac ticket link: https://trac.osgeo.org/osgeolive/ticket/2198

See this page for reference: https://trac.osgeo.org/osgeolive/wiki/How%20to%20document%20the%20quickstart%20file

### CODE REVIEW

File: doc/quickstart/qgis_quickstart.rst [hunk 1]
```diff
@@ -150,24 +127,17 @@ but instead will read as 'My QGIS project'.
...
 A QGIS project saves layers, styles, map extents and settings for use at a later time.
 This information is stored as a QGIS project file and has an extension of '.qgs' or '.qgz',
 with '.qgz' being the default file format in QGIS going into the future.
 
 Let's open an existing QGIS project and have a look at how it works.
 
-#. Open QGIS. Note that when QGIS is opened a user will be presented with a list of recent projects to chose from. This is a quick way of entering into recent projects.
-#. Go to the 'Project' menu on the menu bar.
-#. Select 'Open'.
+#. Open QGIS. Note that when QGIS is opened you will be presented with a list of recent projects to chose from. This is a quick way of entering into recent projects.
```
Comment (timlinux) [inline doc/quickstart/qgis_quickstart.rst]: This is a quick way of entering into recent projects. -> This is a convenient way to open a project you have used recently.

File: doc/quickstart/qgis_quickstart.rst [hunk 2]
```diff
@@ -222,32 +190,22 @@ raster layer on top of all other layers in the map frame.
...
-For this quick start guide, we will focus on the basics of navigation which can be achieved with only
-five tools. These tools are listed below, with the number corresponding to the toolbar image above to
-show the location on the toolbar.
-
-#. Pan: Allows the map to be dragged 'Panned' to a new location.
-#. Zoom in: Allows the map extent to be zoomed in.
-#. Zoom out: Allows the map to be zoomed out.
-#. Zoom to full extent: Changes the map extent to fit all of the map layers into the map frame.
-#. Zoom to layer: Changes the map extent to fit the extent of the layer highlighted in the layer panel.
+* [1] Pan: Allows the map to be dragged 'Panned' to a new location.
```
Comment (timlinux) [inline doc/quickstart/qgis_quickstart.rst]: 'Panned' -> ('panned')

File: doc/quickstart/qgis_quickstart.rst (hunk 1 above)
Comment (flicstar) [reply doc/quickstart/qgis_quickstart.rst]: done

File: doc/quickstart/qgis_quickstart.rst (hunk 2 above)
Comment (flicstar) [reply doc/quickstart/qgis_quickstart.rst]: done

**Comment by flicstar** [comment]
@timlinux Are you happy with the actions I took on your suggestions? Can this be merged now? Cameron and I are wrapping up these PRs in readiness for the docs-only release early next year. Thanks

**Comment by camerons** [comment]
Merging the pull request based on review from @timlinux and follow up actions from @flicstar.

**Comment by timlinux** [comment]
Thank you @flicstar

**Pull request merged**